services/media/service/scripts/setup.ts

```diff
@@ -46,6 +46,10 @@ async function main(): Promise<void> {
         serviceId: 'ax-image-service',
         permissions: ['IMAGE_TYPES_DECLARE'],
       },
+      {
```

Comment:
I'd mention in the PR description that `yarn setup` should be re-run in development and in deployed environments configured Service Account should be adjusted to include an extra permission, as a migration step to enable this functionality.

Reply:
Pull request description was updated to contain migration steps.